File: SpringbootAngular/frontend/src/model/order.ts
import {User} from './model.user';

export class Order {
  id: number;
  note: string;
  orderCode: string;
  lastName: string;
  firstName: string;
  address: string;
  phoneNumber: string;
  email: string;
  createDate: string;
  payment: number;
  userId: number;
  nameOrder: string;

  constructor(id?: number, note?: string, orderCode?: string,
              lastName?: string, firstName?: string,
              address?: string, phoneNumber?: string, email?: string, createDate?: string, payment?: number, userId?: number,
              nameOrder?: string) {
    this.id = id;
    this.note = note;
    this.orderCode = orderCode;
    this.lastName = lastName;
    this.firstName = firstName;
    this.address = address;
    this.phoneNumber = phoneNumber;
    this.email = email;
    this.createDate = createDate;
    this.payment = payment;
    this.userId = userId;
    this.nameOrder = nameOrder;
  }

}

File: SpringbootAngular/backend/src/main/java/com/spring/angular/model/Order.java
package com.spring.angular.model;

import javax.persistence.*;
import java.util.Date;

@Entity
@Table(name = "order")
public class Order {

    private Long id;
    private String orderCode;
    private Long userId;
    private Date createDate;
    private String note;
    private String namerOder;
    private String address;
    private String phoneNumber;
    private String email;
    private int payment;

    @Id
    @GeneratedValue
    @Column(name="id", nullable = false)
    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    @Column(name = "order_code")
    public String getOrderCode() {
        return orderCode;
    }

    public void setOrderCode(String orderCode) {
        this.orderCode = orderCode;
    }

    @Column(name = "user_id")
    public Long getUserId() {
        return userId;
    }

    public void setUserId(Long userId) {
        this.userId = userId;
    }

    @Column(name = "create_date")
    public Date getCreateDate() {
        return createDate;
    }

    public void setCreateDate(Date createDate) {
        this.createDate = createDate;
    }

    @Column(name = "note")
    public String getNote() {
        return note;
    }

    public void setNote(String note) {
        this.note = note;
    }

    @Column(name = "name_order")
    public String getNamerOder() {
        return namerOder;
    }

    public void setNamerOder(String namerOder) {
        this.namerOder = namerOder;
    }

    @Column(name = "address")
    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    @Column(name = "phone_number")
    public String getPhoneNumber() {
        return phoneNumber;
    }

    public void setPhoneNumber(String phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

    @Column(name = "email")
    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    @Column(name = "payment")
    public int getPayment() {
        return payment;
    }

    public void setPayment(int payment) {
        this.payment = payment;
    }
}
